src/pet.js

```diff
@@ -38,6 +39,7 @@ Pet.prototype.feed = function() {
 }
 
 Pet.prototype.checkUp = function() {
+    // no line break after opening block of code
```

Comment:
Just a minor one here, but you shouldn't ever have empty lines after a block of code opens, or before a block of code closes. The indentation should be enough for code readability.